packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -198,22 +198,22 @@
       color: #fff;
 
       &:hover {
-        background-color: darken($datepicker-selected-color, 5%);
+        background-color: $datepicker-selected-color;
       }
     }
 
     &--keyboard-selected {
-      background-color: lighten($datepicker-selected-color, 10%);
+      background-color: $datepicker-selected-color;
       border-radius: $datepicker-border-radius;
       color: #fff;
 
       &:hover {
-        background-color: darken($datepicker-selected-color, 5%);
+        background-color: $datepicker-selected-color;
       }
     }
 
     &--in-selecting-range:not(&--in-range) {
-      background-color: rgba($datepicker-selected-color, 0.5);
+      background-color: $datepicker-selected-color;
```

Comment:
Selecting a range of date is currently not supported but this is not being used today. Maybe we should leave it the way it was instead of changing it since updating it would have the same background color as a single selected date. We can come back to this if we ever allow range selection.

Reply:
Reverted in 7802d23